### PR TITLE
fix(mobile): doc property styles

### PR DESCRIPTION
**Pull request by pengx17**
fix AF-1582
fix AF-1671

- mobile doc info dialog styles
- added ConfigModal for editing property values in modal, including:
  - workspace properties: text, number, tags
  - db properties: text, number, label, link

### CODE REVIEW

**Comment by pengx17** [comment]
* **#8760** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8760?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/component/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     ".": "./src/index.ts",
     "./theme": "./src/theme/index.ts",
     "./ui/*": "./src/ui/*/index.ts",
+    "./mobile": "./src/ui/mobile/index.ts",
```
Comment (EYHN) [inline packages/frontend/component/package.json]: I think we should move the business components into `core`

File: packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx [hunk 2]
```diff
@@ -215,12 +216,12 @@ const notFound = (
...
+const MobileDetailPage = ({
   pageId,
   date,
 }: {
   pageId: string;
-  date: string;
+  date?: string;
 }) => {
   const journalService = useService(JournalService);
   const { openJournal } = useJournalRouteHelper();
```
Comment (CatsJuice) [inline packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx:227]: I separated Page/Journal mainly for avoiding doing this heavy calculation for none journal page.

**Review by CatsJuice — APPROVED** [review]

File: packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx (hunk 2 above)
Comment (pengx17) [reply packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx:227]: When journal mode is toggled the whole page will get remounted. I think the calculation should be muted when mode is a normal doc.

**Pull request merged**